OpenRA.Mods.Common/Traits/World/CrateSpawner.cs

```diff
@@ -20,6 +20,12 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public class CrateSpawnerInfo : ITraitInfo
 	{
+		[Desc("Default value of the crates checkbox in the lobby.")]
+		public readonly bool Enabled = true;
+
+		[Desc("Prevent the enabled state from being changed in the lobby.")]
```

Comment:
That's not obvious from the context, especially when a TraitInfo has more than just these two fields. Please make it more specific, as you already did with many of the other moved options.